web/src/games/barnbarter/definitions.ts

```diff
@@ -0,0 +1,62 @@
+import { Ctx } from 'boardgame.io';
+import { IGameArgs } from 'gamesShared/definitions/game';
+
+export interface IBoardState {
+  showCallTable: boolean;
+}
+
+export interface IBoardProps {
+  G: IG;
+  ctx: Ctx;
+  moves: any;
+  playerID: string;
+  isActive: boolean;
+  gameArgs?: IGameArgs;
+  events?: any;
+}
+
+export interface ICard {
+  name: string;
```

Comment:
Probably using an enum is better, since we can only have few types of animals?

Reply:
True. But I think its fine like this. I expect no updates to the game logic, only to frontend.